pkg/cache/cache.go

```diff
@@ -0,0 +1,132 @@
+package cache
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/aquasecurity/trivy/pkg/iac/state"
+)
+
+type Cache struct {
```

Comment:
Is this cache specific to AWS only, or can it be used for other cloud providers in the future?

Reply:
For now just AWS. If we ever decide to add other cloud providers, we can reconsider it.